index.html

```diff
@@ -2366,120 +2366,130 @@ <h2>Services</h2>
-<a>representation</a> of a <a>DID document</a> via the <code>contentType</code> <a>DID
-resolver</a> metadata field (see <a href="#did-resolution"></a>), <em>not</em>
-through the content of the <a>DID document</a> alone.
-    </p>
-
-    <p>
-A <a>conforming producer</a>
-MUST NOT produce a <a>DID document</a> containing different entries with
-the same <code>id</code> value. A <a>conforming consumer</a> MUST produce
-an error if it detects different entries with the same <code>id</code> value.
-    </p>
+    <section>
+      <h2>Production and Consumption</h2>
 
-    <p>
-Unrecognized entries in the <a href="#data-model">data model</a> MUST be
-preserved. An unrecognized entry is any entry that does not have explicit
-processing rules known to the consumer or producer. Consumers MUST add all
-entries that do not have explicit processing rules for the
-<a>representation</a> being consumed to the <a href="#data-model">data
-model</a> using only the <a>representation</a>'s generic type processing rules.
-Producers MUST serialize all entries in the <a href="#data-model">data
-model</a> that do not have explicit processing rules for the
-<a>representation</a> being produced using only the <a>representation</a>'s
-generic type processing rules.
-    </p>
+      <p>
+In addition to the <a>representations</a> defined in this specification,
+implementers can use other <a>representations</a>, providing each such
+<a>representation</a> is properly specified (including rules for
+interoperable handling of properties not listed in the DID Specification
+Registries [[?DID-SPEC-REGISTRIES]]). See <a href="#extensibility"></a>
+for more information.
+      </p>
+      <p>
+The requirements for all <a>representations</a> are as follows:
+      </p>
+      <ol>
+        <li>
+A <a>representation</a> MUST define deterministic production and consumption
+rules for all data types specified in <a href="#data-model"></a>.
+        </li>
+        <li>
+A <a>representation</a> MUST be uniquely associated with an IANA-registered
+MIME type.
+        </li>
+        <li>
+A <a>representation</a> MUST define fragment processing rules for its MIME
+type that are conformant with the fragment processing rules defined in
+<a href="#fragment"></a>.
+        </li>
+        <li>
+A <a>representation</a> SHOULD use the lexical representation of <a
+href="#data-model">data model</a> data types. For example, JSON and JSON-LD use
+the XML Schema <code>dateTime</code> lexical serialization to represent
+<a>datetimes</a>. A <a>representation</a> MAY choose to serialize the <a
+href="#data-model">data model</a> data types using a different lexical
+serializations as long as the <a>consumption</a> process back into the <a
+href="#data-model">data model</a> is lossless. For example, some CBOR-based
+<a>representations</a> express <a>datetime</a> values using integers to
+represent the number of seconds since the Unix epoch.
```

Comment:
`some CBOR-based representations express data time values using integers` 

I don't believe this is correct.  We constrained the CBOR to major type 3, which is string.   I think this type of extensibility could be made in the DID-Spec Registries, not in Core.

Reply:
The CBOR-LD encoding of datetime uses an integer.